packages/server/README.md

```diff
@@ -60,6 +60,10 @@ Creates a new tus server with options.
 
 The route to accept requests (`string`).
 
+#### `options.maxSize`
+
+Max file size (in bytes) allowed when uploading (`number`)
```

Comment:
This needs documentation that `maxSize` can also be a function.

Reply:
Done